### PR TITLE
Splitting project into modules

### PR DESCRIPTION
**Pull request by mfvanek**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,243 +1,232 @@
...
+        ignoreFailures = false
+        maxWarnings = 0
+        maxErrors = 0
     }
-}
-
-javadoc {
-    if (JavaVersion.current().isJava9Compatible()) {
-        options.addBooleanOption('html5', true)
+//    checkstyleMain {
```
Comment (Evreke) [inline build.gradle:85]: Better to remove if it's not needed. If this is necessary for some reason, then leave an explanatory comment.

File: pg-index-health/build.gradle [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    testImplementation project(':pg-index-health-testing')
+    testImplementation(testFixtures(project(":pg-index-health-model")))
+    testImplementation(testFixtures(project(":pg-index-health-jdbc-connection")))
+    testImplementation 'org.junit.jupiter:junit-jupiter-params'
+    testImplementation "ch.qos.logback:logback-classic:${logbackVersion}"
+    testImplementation "org.mockito:mockito-core:${mockitoVersion}"
+    testImplementation "org.awaitility:awaitility:${awaitilityVersion}"
+    testImplementation "org.apache.commons:commons-lang3:${commonsLang3Version}"
+
+    //testImplementation "org.apache.commons:commons-dbcp2:${dbcp2Version}"
```
Comment (Evreke) [inline pg-index-health/build.gradle:15]: If it's done intentionally, it's useful to share this knowledge with others.

**Review by Evreke — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (mfvanek) [reply build.gradle]: Fixed

File: pg-index-health/build.gradle (hunk 2 above)
Comment (mfvanek) [reply pg-index-health/build.gradle]: Fixed

**Pull request merged**